plugins/woocommerce/client/admin/client/settings-payments/components/other-payment-methods.tsx

```diff
@@ -53,27 +52,73 @@ const usePaymentGatewayData = () => {
 	}, [] );
 };
 
-const AdditionalGatewayImages = ( {
-	additionalGateways,
-}: {
-	additionalGateways: PaymentGateway[];
-} ) => (
-	<>
-		{ additionalGateways.map( ( gateway ) => (
-			<img
-				key={ gateway.id }
-				src={ gateway.image_72x72 }
-				alt={ gateway.title }
-				width="24"
-				height="24"
-				className="other-payment-methods__image"
-			/>
-		) ) }
-		{ _x( '& more.', 'More payment providers to discover', 'woocommerce' ) }
-	</>
-);
+// TODO implement gateway images.
+// const AdditionalGatewayImages = ( {
+// 	additionalGateways,
+// }: {
+// 	additionalGateways: PaymentGateway[];
+// } ) => (
+// 	<>
+// 		{ additionalGateways.map( ( gateway ) => (
+// 			<img
+// 				key={ gateway.id }
+// 				src={ gateway.image_72x72 }
+// 				alt={ gateway.title }
+// 				width="24"
+// 				height="24"
+// 				className="other-payment-methods__image"
+// 			/>
+// 		) ) }
+// 		{ _x( '& more.', 'More payment providers to discover', 'woocommerce' ) }
+// 	</>
+// );
 
 export const OtherPaymentMethods = () => {
```

Comment:
Similar comment to above, I think something like `AlternativePaymentMethods` or `AdditionalPaymentMethods` could be a good component name?

Reply:
Good point. I think "Other" is probably what we want to use then. Because it looks like the providers here are competing ones that we want to give the user the choice to use if they prefer to.

Reply:
Also, following our discussion on Slack @oaratovskyi @vladolaru we should avoid using _methods_ in the naming and use _gateways_ (or _extensions_) here. I think in this case they are probably interchangeable (since they are likely to be both), but maybe gateway is better for consistency.

Reply:
fixed in https://github.com/woocommerce/woocommerce/pull/52030/commits/0444c8c78b346bc2b8061a88e4e8898d5f3357f7!